plugins/PrivacyManager/angularjs/opt-out-customizer/opt-out-customizer.directive.html

```diff
@@ -44,9 +44,9 @@ <h3>HTML code to embed on your website</h3>
     <pre piwik-select-on-focus>&lt;iframe
         style="border: 0; height: 200px; width: 600px;"
         src="{{ optOutCustomizer.iframeUrl }}"
-        ></iframe></pre>
+        &gt;&lt;/iframe&gt;</pre>
     <p ng-bind-html="'CoreAdminHome_OptOutExplanationIntro'|translate:'&lt;a  href=\'' + optOutCustomizer.iframeUrl + '\' rel=\'noreferrer\' target=\'_blank\'>':'&lt/a>'">
```

Comment:
I think there is a `;` missing at the end of the entity.

Reply:
right, didn't notice